addons/website/views/snippets.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 <template id="s_cover" name="Cover">
-    <section class="s_text_block_image_fw oe_img_bg oe_custom_bg" style="background-image: url(/web/image/website.s_cover_default_image); background-position: 50% 100%">
+    <section class="s_text_block_image_fw oe_img_bg" style="background-image: url(/web/image/website.s_background_image_03); background-position: 50% 100%">
```

Comment:
in snippet select used default picture when open the menu for the first time, but if we use website.s_cover_default_image then it will not identify from the list so in options no-one will selected. in cover snippet to select the default used image i changed this